src/grid/OneOnOneLayout.tsx

```diff
@@ -0,0 +1,132 @@
+    const spotlightAlignmentValue = useObservableEagerState(spotlightAlignment);
+
+    const [generation] = useReactiveState<number>(
+      (prev) => (prev === undefined ? 0 : prev + 1),
+      [width, height, model.spotlight === undefined, spotlightAlignmentValue],
+    );
+
+    const spotlightTileModel: SpotlightTileModel | undefined = useMemo(
+      () =>
+        model.spotlight && {
+          type: "spotlight",
+          vms: model.spotlight,
+          maximised: false,
+        },
+      [model.spotlight],
+    );
+
+    const onDragSpotlight: DragCallback = useCallback(
+      ({ xRatio, yRatio }) =>
+        spotlightAlignment.next({
+          block: yRatio < 0.5 ? "start" : "end",
+          inline: xRatio < 0.5 ? "start" : "end",
+        }),
+      [],
+    );
+
+    return (
+      <div ref={ref} data-generation={generation} className={styles.layer}>
+        {spotlightTileModel && (
+          <Slot
+            className={classNames(styles.slot, styles.spotlight)}
+            id="spotlight"
+            model={spotlightTileModel}
+            onDrag={onDragSpotlight}
+            data-block-alignment={spotlightAlignmentValue.block}
+            data-inline-alignment={spotlightAlignmentValue.inline}
+          />
+        )}
+      </div>
+    );
+  }),
+
+  scrolling: forwardRef(function OneOnOneLayoutScrolling({ model, Slot }, ref) {
+    const { width, height } = useObservableEagerState(minBounds);
+    const pipAlignmentValue = useObservableEagerState(pipAlignment);
+    const { tileWidth, tileHeight } = useMemo(
+      () => arrangeTiles(width, height, 1),
+      [width, height],
+    );
+
+    const [generation] = useReactiveState<number>(
+      (prev) => (prev === undefined ? 0 : prev + 1),
+      [width, height, pipAlignmentValue],
+    );
+
+    const remoteTileModel: GridTileModel = useMemo(
+      () => ({ type: "grid", vm: model.remote }),
+      [model.remote],
+    );
+    const localTileModel: GridTileModel = useMemo(
+      () => ({ type: "grid", vm: model.local }),
+      [model.local],
+    );
+
+    const onDragLocalTile: DragCallback = useCallback(
+      ({ xRatio, yRatio }) =>
+        pipAlignment.next({
+          block: yRatio < 0.5 ? "start" : "end",
+          inline: xRatio < 0.5 ? "start" : "end",
+        }),
+      [],
+    );
+
+    return (
+      <div ref={ref} data-generation={generation} className={styles.layer}>
+        <Slot
+          id={remoteTileModel.vm.id}
+          model={remoteTileModel}
+          className={styles.container}
+          style={{ width: tileWidth, height: tileHeight }}
+        >
+          <Slot
+            className={classNames(styles.slot, styles.local)}
+            id={localTileModel.vm.id}
+            model={localTileModel}
+            onDrag={onDragLocalTile}
+            data-block-alignment={pipAlignmentValue.block}
+            data-inline-alignment={pipAlignmentValue.inline}
+          />
+        </Slot>
```

Comment:
Interesting. The slot for the local media feed is a child of the remote participant slot?
That makes actually.
So the local tile is always inside the parent/remote tile. Will this also hold true if we have a portrait remote media feed on a landscape screen. in that case i can imagine we would like the local tile to be outside the remote tile?
(currently we use the "show grey borders" (fit to frame) approach but in theory it would be nice to have the tile be the correct aspect ratio already)